boswatch/inputSource/sdrInput.py

```diff
@@ -36,6 +36,8 @@ def _runThread(self, dataQueue, sdrConfig, decoderConfig):
             sdrProc.addArgument("-p " + str(sdrConfig.get("error", default="0")))      # frequency error in ppm
             sdrProc.addArgument("-l " + str(sdrConfig.get("squelch", default="1")))    # squelch
             sdrProc.addArgument("-g " + str(sdrConfig.get("gain", default="100")))     # gain
+            if (sdrConfig.get("fir_size") is not None):
```

Comment:
to discuss:
`if sdrConfig.get("fir_size", default=None):`

`is not None` is a double negation.